Formula/l/lua-language-server.rb

```diff
@@ -49,7 +49,12 @@ def install
     BASH
   end
 
+  def post_install
```

Comment:
Sorry, I forgot it would be nice to add a comment why we do `post_install` here, something like
```suggestion
  # `lua-language-server` uses `changelog.md` in `libexec`
  # directory to determine its version. Installing the changelog
  # in `def install` does not work because it cleanups metafiles
  # from non-root directory
  def post_install
```

Reply:
OK